py/circuitpy_mpconfig.h

```diff
@@ -179,17 +179,11 @@ typedef long mp_off_t;
 // board-specific definitions, which control and may override definitions below.
 #include "mpconfigboard.h"
 
-// CIRCUITPY_FULL_BUILD is defined in a *.mk file.
-
+// CIRCUITPY_FULL_BUILD is defined in py/circuitpy_mpconfig.mk
```

Comment:
The point of the original (not well written) comment is that the default value of `CIRCUITPY_FULL_BUILD` could be overriden in a .mk file. Right now there's no example of that.

Reply:
You could just remove the comment now.

Reply:
I honestly think the whole system is a little dangerous, because it has two separate flags that do the same thing (SMALL_BUILD = !FULL_BUILD) and you could easily do something like setting FULL_BUILD=0, SMALL_BUILD=0 without realizing. It might be better if we remove SMALL_BUILD altogether, but I didn't want to go overboard.

Reply:
`CIRCUITPY_SMALL_BUILD` is used in `atmel-samd` for 192kB builds (and only in that port). So it could probably be removed from here, but something equivalent still needs to be in `atmel-samd`.

Reply:
Right, what I'm saying is that it might be better if atmel-samd 192kB builds did this by setting CIRCUITPY_FULL_BUILD=0, rather than using SMALL_BUILD which literally just does the same thing but with an extra step.